src/main/java/duke/io/FileIO.java

```diff
@@ -0,0 +1,50 @@
+package duke.io;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class FileIO {
+  private String filepath;
+  private String filename;
+  private String description;
+
+  public FileIO(String filepath, String filename, String description) {
+    this.filepath = filepath;
+    this.filename = filename;
+    this.description = description;
+  }
+
+  public String read() throws FileNotFoundException {
+    File f = new File(filepath + filename);
+    Scanner in = new Scanner(f);
+    StringBuilder sb = new StringBuilder();
+    while (in.hasNext()) {
+      sb.append(in.nextLine().trim());
+      sb.append('\n');
+    }
+    return sb.toString().trim();
+  }
```

Comment:
inputFormat and exampleDatetime should be made final constants since they are not meant to change.